assets/performanceprofile/scripts/add-offline-cpus.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
+
+set -euo pipefail
+
+disable_cpu="0"
```

Comment:
I think "0" and "1" are idiomatic in bash, so this is a bit of overdoing

Reply:
done